### DISCUSSION TITLE
Username / password to use for google auth accounts

### DISCUSSION THREAD

**Discussion by lgilson-private** [tolwi/hassio-ecoflow-cloud#274] [General]
Hi,

I created my ecoflow account by logging in with google. It didn't ask for a password.

What username and password do I need to enter for this plugin?

Thanks

**Comment by Nid01** [comment]
What options do you see, when accessing "Profile", click on the profile icon in the upper left corner?
There you should see your e-mail address and maybe be able to change your password.
When you can't change your password you maybe can set on by resetting it via the login site of EcoFlow since I don't know if it will be possible to log into the API for this integration via Google or Apple accounts.

**Reply by lgilson-private** [reply]
I see my Google account as email and options:

- unlink ("You will no longer be able to logon using this social account")
- reset password. Reset password requires the old password and a new password. It doesn't work with blank.

**Reply by Nid01** [reply]
When you are in the app than there should be the option "change password" and not "reset password".
Reset password never requires knowledge of the old password, otherwise it wouldn't be a reset.
Maybe you have luck there: https://account.ecoflow.com/forget
Otherwise I guess you need to register your account without Google.

**Reply by lgilson-private** [reply]
Thanks for the quick help.

Going the forgot-password way let's you add a password to a Google Auth linked account. It's enough to get the ecoflow stuff running.
